### ISSUE TITLE
For Gateway M1S CN, does it only work for Mi Home Mode? How about aqara home mode?

### ISSUE THREAD

**Issue by Peek-A-Booo** [niceboygithub/AqaraGateway#78]
my curtains c2 can only add in aqara home app. can i use this component to control it from home assistant

**Comment by shift-del1** [comment]
Yes, if you flash the modified firmwre, you can switch to Aqara Home mode by pressing the button 10 times, and when the device confirms the reset, you have to press it twice rapidly. Modified firmware retains telnet, so it starts inAH mode too. With this running, you can add the gateway to HA too (will work parallel with AH).
